Source/JavaScriptCore/ftl/FTLOSRExitCompiler.cpp

```diff
@@ -624,7 +624,7 @@ static void compileStub(VM& vm, unsigned exitID, JITCode* jitCode, OSRExit& exit
         patchBuffer, OSRExitPtrTag, nullptr,
         "FTL OSR exit #%u (D@%u, %s, %s) from %s, with operands = %s",
             exitID, exit.m_dfgNodeIndex, toCString(exit.m_codeOrigin).data(),
-            exit.m_kind, toCString(*codeBlock).data(),
+            toCString(exit.m_kind).data(), toCString(*codeBlock).data(),
```

Comment:
Does this not work? I thought I tested it when I uploaded my enum to string patch...

Reply:
Unfortunately, the old code crashes.